src/styles/variables.css

```
@@ -0,0 +1,18 @@
+:root {
+  --main-font-color: #060b35;
+  --secondary-font-color: #616070;
+
+  --button-color: #00acdc;
+
+  --font-size-small: 10px;
+  --font-size-medium: 12px;
+
+  --font-height-medium: 14px;
+  --font-height-large: 18px;
+
+  --font-weight-bold: 700;
+  --font-weight-normal: 400;
+
+  --horizontal-spacing: 16px;
+  --vertical-spacing: 16px;
```

You could merge those 2 as they have the same value
```suggestion
  --spacing: 16px;
```